### ISSUE TITLE
introduce Molecular Event class to GOPlus

### ISSUE THREAD

**Issue by goodb** [geneontology/go-ontology#19960]
Following discussion in the Pathways2GO project - https://github.com/geneontology/pathways2GO/issues/98 , we would like to add a new class into the upper levels of the GO world called 'Molecular Event'.  Specific definition forthcoming, but in general it is meant to handle cases where things are happening in the cell but no specific enabling molecule can be defined.  

ping @cmungall @deustp01 @thomaspd @ukemi @vanaukenk

**Comment by pgaudet** [comment]
@ukemi @deustp01 is this still planned?

**Comment by deustp01** [comment]
> It was done in some way.

Or did we hack around this by using the root GO molecular function for the enabler? An item for weeds discussion. @vanaukenk

**Comment by ukemi** [comment]
I don't think we hacked around it using the root MF. They are definitely there in the models:
http://noctua.geneontology.org/editor/graph/gomodel:R-HSA-9634635
but they might be there only in Reacto:
obo:go/extensions/reacto.owl#molecular_event] Molecular Event

**Comment by pgaudet** [comment]
@ukemi In Reactome, this serves to describe complex formation, for example, where there is a Reactome reaction but for which we cannot identify a MF and an enabler. The GO-Reactome project is trying to get rid of them (sometimes by capturing the enabler, which often exists but has not always been captured) 

@deustp01 In GO there is no way to capture association/dissociation , but there are cases when Reactome wants to capture a causal connection between two events.

@ukemi suggests this should NOT be in GO-plus.

**Comment by ukemi** [comment]
I think if this class is in any model that we eventually make production, then we have to include it in GO-plus.
